src/lua/functions/creatures/player/player_functions.cpp

```diff
@@ -391,6 +391,16 @@ int PlayerFunctions::luaPlayerRemoveTaskHuntingPoints(lua_State* L) {
 	return 1;
 }
 
+int PlayerFunctions::luaPlayerGetTaskHuntingPoints(lua_State* L) {
+	// player:getTaskHuntingPoints()
+	if (const Player* player = getUserdata<Player>(L, 1)) {
+		lua_pushnumber(L, static_cast<double>(player->getTaskHuntingPoints()));
+	} else {
+		lua_pushnil(L);
+	}
```

Comment:
Can we report a specific error instead of nil:
```suggestion
	const Player* player = getUserdata<Player>(L, 1)
	if (player == nullptr) {
		reportErrorFunc(getErrorDesc(LUA_ERROR_PLAYER_NOT_FOUND));
		pushBoolean(L, false);
		return 1;
	}

	lua_pushnumber(L, static_cast<double>(player->getTaskHuntingPoints()));
```